eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl

```diff
@@ -22,13 +22,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
-<!--
-  @todo #1890:30m Change the behaviour of conversions for names
```

Comment:
@maxonfjvipon Why do you remove that puzzle in that PR? Is it related issue?

Reply:
@volodya-lombrozo @Graur please explain

Reply:
@volodya-lombrozo I asked @maxonfjvipon to remove this puzzle by your [suggestion](https://github.com/objectionary/eo/issues/2096#issuecomment-1677006145) and because I couldn't find a test example of the related improvement. Right now I can't remove this puzzle by myself.

Reply:
@Graur Ok, thanks for the explanation. I believe next time it's better to create a separate PR for that.